### PR TITLE
Add support for Mi version of light detector (lumi.sen_ill.mgl01)

### PR DESCRIPTION
**Pull request by SwoopX**
See #4648

Tested and confirmed to be working.

### CODE REVIEW

File: de_web_plugin.cpp [hunk 1]
```diff
@@ -77,6 +77,7 @@ const quint64 macPrefixMask       = 0xffffff0000000000ULL;
 const quint64 legrandMacPrefix    = 0x0004740000000000ULL;
 const quint64 dishMacPrefix       = 0x0008890000000000ULL;
 const quint64 ikeaMacPrefix       = 0x000b570000000000ULL;
+const quint64 danalockMacPrefix   = 0x000b570000000000ULL; // note: same as ikeaMacPrefix
```
Comment (manup) [inline de_web_plugin.cpp:80]: Should we just keep one and rename it to `silabs1MacPrefix`?

https://maclookup.app/search/result?mac=000b570

**Review by manup — COMMENTED** [review]
Question for the mac prefix, we have the huge opportunity to remove *one line* from the code base :smile:

File: de_web_plugin.cpp (hunk 1 above)
Comment (SwoopX) [reply de_web_plugin.cpp:80]: You overlooked the other duplicate entry ;)

I'd prefer to leave it here as is and have a seperate PR for that as we need to propagate the changes through all files.

Comment (manup) [reply de_web_plugin.cpp:80]: Sounds good

**Pull request merged**